<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Competitive Coevolution GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>821</width>
      <height>781</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="networkSettingsTab">
     <attribute name="title">
      <string>System Settings</string>
     </attribute>
     <widget class="QPushButton" name="connectButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="portEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>7474</string>
      </property>
     </widget>
     <widget class="QLabel" name="hostNameLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hostname:</string>
      </property>
     </widget>
     <widget class="QLabel" name="portLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="hostNameEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>localhost</string>
      </property>
     </widget>
     <widget class="QPushButton" name="disconnectButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>20</y>
        <width>141</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Test Message</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="connectionStatus">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Not Connected</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="pauseStatusLine">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>710</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Not Paused</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pausePushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>710</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause System</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitSimButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>710</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit Server Simulation</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="generationDataTab">
     <attribute name="title">
      <string>Generation Data</string>
     </attribute>
     <widget class="QLineEdit" name="genCounterLine">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Generation:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="blueTeamGroupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>Blue Team</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>108</width>
         <height>201</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Champ Index:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Neurons:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Fitness Score:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Best Ever
Fitness:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Number Species:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>112</width>
         <height>221</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="blueChampIndexLine">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="blueNumNeuronsLine">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="blueFitnessLine">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="blueBestFitnessLine">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="blueNumSpeciesLine">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="redTeamGroupBox">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>80</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>Red Team</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>108</width>
         <height>201</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Champ Index:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Neurons:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Fitness Score:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Best Ever
Fitness:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Number Species:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>112</width>
         <height>221</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLineEdit" name="redChampIndexLine">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="redNumNeuronsLine">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="redFitnessLine">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="redBestFitnessLine">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="redNumSpeciesLine">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="renderDataTab">
     <attribute name="title">
      <string>Render Data</string>
     </attribute>
     <widget class="QComboBox" name="timeModeComboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maxCount">
       <number>2</number>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContents</enum>
      </property>
     </widget>
     <widget class="QWidget" name="renderWidget" native="true">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>700</width>
        <height>700</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="helpTab">
     <attribute name="title">
      <string>Help</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>621</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Written by Drew Kirkpatrick as part of his M.S. thesis work at</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>621</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>contact: drew.kirkpatrick@gmail.com</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>11</y>
        <width>615</width>
        <height>92</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>This software was written as part of thesis work in competitive coevolution.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>It is a GUI client meant to remotely connect to a commandline only server that runs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>the competitive coevolutionary experiments. Network communication is through</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TCP, and can be port forwarded over ssh for remote Linux computers . </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>621</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Florida Institute of Technology.</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
